pandera/strategies.py

```diff
@@ -50,6 +50,14 @@ def composite(fn):
 IndexComponent = Any
 
 
+def _mask(
+    val: Union[pd.Series, pd.Index], null_mask: List[bool]
+) -> Union[pd.Series, pd.Index]:
+    if pd.api.types.is_timedelta64_dtype(val):
+        return val.where(null_mask, pd.NaT)
```

Comment:
I think this should be `val.mask(null_mask, pd.NaT)`, since a `True` in `null_mask` should be replaced by the second arg (whose default is `nan`): https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.mask.html